scheduler/src/cook/kubernetes/api.clj

```diff
@@ -812,10 +814,12 @@
 (defn set-mem-cpu-resources
   "Given a resources object and a CPU and memory request and limit, update the resources object to reflect the
   desired requests and limits."
-  [^V1ResourceRequirements resources memory-request memory-limit cpu-request cpu-limit]
+  [^V1ResourceRequirements resources memory-request cpu-request cpu-limit & {:keys [memory-limit] :or {memory-limit nil}}]
```

Comment:
I think it's better to revert to the old function signature. see below for what to pass in

Reply:
done